src/Ankimon/ankimon_mobile_web/history.html

```
@@ -57,6 +57,13 @@ <h1>MOBILE</h1>
                 <span class="nav-menu-desc">Configure Ankimon</span>
               </div>
             </button>
+            <button class="nav-menu-item" role="menuitem" data-screen="monthly">
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)